GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -91,16 +91,13 @@ public override NamedPipeServerStream CreatePipeByName(string pipeName)
 
         public override IEnumerable<EventListener> CreateTelemetryListeners(string providerName, string enlistmentId, string mountId)
         {
-            // TODO: return TelemetryDaemonEventListener when the telemetry daemon has been implemented for Mac
+            string gitBinRoot = this.GitInstallation.GetInstalledGitBinPath();
 
-            // string gitBinRoot = this.GitInstallation.GetInstalledGitBinPath();
-            // var daemonListener = TelemetryDaemonEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId, pipeName: "vfs");
-            // if (daemonListener != null)
-            // {
-            //     yield return daemonListener;
-            // }
-
-            yield break;
+            TelemetryDaemonEventListener daemonListener = TelemetryDaemonEventListener.CreateIfEnabled(gitBinRoot, providerName, enlistmentId, mountId);

```

Comment:
It looks like this code is identical to the code in `GVFSPlatform`.

Assuming that's the case, `CreateTelemetryListeners` can be removed from `GVFSPlatform` (`GVFSPlatform` is intended for code that needs to be platform specific).